ServiceWebsite/ServiceWebsite.AcceptanceTests/Features/SelfTest/TestYourEquipment.feature

```diff
@@ -22,23 +22,8 @@ Scenario: Test Your Equipment Representative
 	When the Test Your Equipment video has ended
 	And the user clicks the Watch the video again button
 	Then the Test Your Equipment video begins to play
+	When the Test Your Equipment video has ended
```

Comment:
You need to add this to the 'Scenario: Test Your Equipment Individual' test as well

Reply:
Fixed this.